Vic2ToHoI4/Source/OutHoi4/Modifiers/OutDynamicModifiers.cpp

```diff
@@ -0,0 +1,35 @@
+#include "OutDynamicModifiers.h"
+#include "HOI4World/Modifiers/DynamicModifiers.h"
+#include "HOI4World/Modifiers/Modifier.h"
+#include "OSCompatibilityLayer.h"
+#include <fstream>
+
+namespace HoI4
+{
+
+void outDynamicModifiers(const DynamicModifiers& dynamicModifiers, const Configuration& theConfiguration)
+{
+	if (!commonItems::TryCreateFolder("output/" + theConfiguration.getOutputName() + "/common/dynamic_modifiers"))
+	{
+		Log(LogLevel::Error) << "Could not create output/" + theConfiguration.getOutputName() +
```

Comment:
Throw an exception instead of directly logging and exiting. The outermost try/catch will handle it.